src/common/interfaces/workos-options.interface.ts

```diff
@@ -1,6 +1,9 @@
+import { AppInfo } from './app-info.interface';
+
 export interface WorkOSOptions {
   apiHostname?: string;
   https?: boolean;
   port?: number;
   config?: RequestInit;
+  appInfo?: AppInfo;
```

Comment:
This is something for internal use only with the Next SDK, hence why I didn't add docs. I named it `appInfo` in case we want to expand it in the future to contain information about apps that use WorkOS. See this [Stripe example](https://docs.stripe.com/building-plugins?lang=node#setappinfo) for inspiration.

Reply:
Makes sense if there's already some prior art.